cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -398,6 +398,21 @@ public String parseLogging(Logging logging, ConfigMap externalCm) {
         }
     }
 
+    /**
+     * Adds 'monitorInterval=5' to external logging ConfigMap. If ConfigMap already has this value, it is persisted.
+     * This property is ignored by log4j but used by log4j2.
+     * @param data
+     * @return
+     */
+    protected String addMonitorIntervalToExternalLogging(String data) {
+        if (!data.contains("monitorInterval=")) {
+            // do not override custom value
+            return data + "\nmonitorInterval=5";
```

Comment:
How can the users change this? Through the logger configuration in the CR? Or only with external logging configuration? In general, 5 seconds (is it seconds?) seems very short ... logging is not changed so often ... would 30 seconds or 1 minute be ore suitable?

Reply:
I do not think it is crucial to make it changeable by user. Basically it should be `<5, reconciliationPeriod>`. 5 seconds is the minimum from log4j2. Setting it to low value will have effect she logging will be updated shortly after the reconciliation. Some bigger value will take a long time to reload new configuration although there will be less reloads from log4j2.

Reply:
Have you done any tests how much does the reloading cost? I think 5 seconds is quite agressive given how much often would this happen. If the costs are negligible it is fine. If not, we should use some higher value I think.

Reply:
No I did not. I increased the value to 30s.